Tests/FunctionalTests/Sources/FTRScrollViewTest.m

```diff
@@ -28,7 +28,7 @@ - (void)setUp {
   [self openTestViewNamed:@"Scroll Views"];
 }
 
-- (void)testScrollToTopEdge {
+- (void)DISABLED_testScrollToTopEdge {
```

Comment:
You should add another test so that it doesn't disable an existing test that's providing us with indication that scroll to top edge works for some scenarios.